lib/metric_fu/metrics/flog/generator.rb

```diff
@@ -1,5 +1,6 @@
 require "pathname"
 require "optparse"
+require "path_expander"
```

Comment:
I'm sort of only using it because I'm mimicing the flog api... have mixed feelings